ppl/zqd/ztests/cluster/count.yaml

```
@@ -0,0 +1,9 @@
+tag: cluster
+script: |
+  # Test assumes a remote zqd instance with space "files" is available at 9867
+  time zapi -s files get -workers 4 -t 'count()' 2>&1
+outputs:
```

Nit: Put a blank line between top level keys like we do in all the other ztests.
```suggestion
tag: cluster

script: |
  # Test assumes a remote zqd instance with space "files" is available at 9867
  time zapi -s files get -workers 4 -t 'count()' 2>&1

outputs:
```

---

(And do this for the other ztests you're adding here too.)